### PR TITLE
Space Chatter

### PR DESCRIPTION
**Pull request by Arokha**
Adds random space traffic control chatter during the game. It's rare, not spammy. Like every 35 minutes it has one short conversation.

It adds a bunch of lore datums about different parties in space and ship names and stuff. That can be used for other stuff later.

If you find it annoying, you can disable it at the comms consoles (don't need to log in, anyone can do it).

***Normal traffic***
[Common] SCV Falken states, "Virgo TraCon, this is Federation SCV Falken on a military route to the Gateway 98-C at Arest, requesting refueling information."
[Common] Virgo Air/Space Control states, "Federation SCV Falken, this is Virgo TraCon, sending refueling information now."
[Common] SCV Falken states, "Virgo TraCon, thank you, good day."

***5% chance of wrong frequency***
[Common] SDV Lucina states, "Sol Command, this is Federation SDV Lucina on a diplomatic flight to NSB Checkmate, requesting remote telemetry data."
[Common] Virgo Air/Space Control states, "Federation SDV Lucina, this is Virgo TraCon, wrong frequency. Switch to 855.2."
[Common] SDV Lucina states, "Virgo TraCon, understood, apologies."

***2% chance of emergency traffic***
[Common] UNSC Colorado states, "This is Navy UNSC Colorado declaring an emergency! We have unknown life forms on board!"
[Common] Virgo Air/Space Control states, "Navy UNSC Colorado, this is Virgo TraCon, copy. Switch to emergency responder channel 639.2."
[Common] UNSC Colorado states, "Virgo TraCon, okay, switching now."

### CODE REVIEW

**Comment by JoanRisu** [comment]
Dis neat o3o

**Comment by Screemonster** [comment]
I almost want to say include a (rare) message suggesting that crew are inbound that are intended for _some other_ station and TraCon informing them that they're off course and and probably don't want to dock at Virgo.

**Comment by SpadesNeil** [comment]
We need to mix this up a bit with randomly generated sentences as opposed to just a few static statements--similar to how ion laws are done. However I like the concept of it.

**Comment by JoanRisu** [comment]
I think it kinda does that, it's just the name and type of mission is dynamic but the statement for the type of mission is static.

File: code/modules/busy_space/air_traffic.dm [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+	var/destname = pick(dest.holding_names)			//Pick a random holding from the destination
+
+	var/combined_name = "[owner] [prefix] [shipname]"
+	var/alt_atc_names = list("Virgo TraCon","Virgo Control","Virgo ATC","Virgo Airspace")
+	var/wrong_atc_names = list("Sol Command","Polaris TraCon","Orion Control")
+	var/mission_noun = list("flight","mission","route")
+	var/request_verb = list("requesting","calling for","asking for")
+
+	//First response is 'yes', second is 'no'
+	var/requests = list("Virgo transit clearance" = list("permission for transit granted", "permission for transit denied, contact regional on 953.5"),
```
Comment (JoanRisu) [inline code/modules/busy_space/air_traffic.dm]: Actually, @SpadesNeil, is dis what chu looking for?

**Comment by Arokha** [comment]
It does use randomly generated stuff. There's a list of things people can request, a chance for them to be declined (10%), a chance that they ask on the wrong frequency, and stuff.

`full_request = "[callname], this is [combined_name] on a [mission] [pick(mission_noun)] to [destname], [pick(request_verb)] [request]."`

**Comment by SpadesNeil** [comment]
In my defense, I was at work (though leaving) when I commented. :U No time to look at code.

**Pull request merged**